python/pyspark/ml/regression.py

```diff
@@ -347,6 +347,20 @@ def r2(self):
         """
         return self._call_java("r2")
 
+    @property
+    @since("2.4.0")
+    def r2adj(self):
+        """
+        Returns Adjusted R^2^, the adjusted coefficient of determination.
```

Comment:
`R^2^` is a scaladoc format.  How about just `R^2`? and could you fix in `def r2` also

Reply:
sure, I will change.